src/components/attendee/ProfileModalEditLinks/ProfileModalEditLinks.tsx

```diff
@@ -62,7 +62,7 @@ export const ProfileModalEditLinks: React.FC<ProfileModalEditLinksProps> = ({
       <div className={styles.ProfileModalEditLinks__header}>
         <div>Profile links</div>
         {renderedLinks}
-        <Button onClick={onAddLink} variant="alternativeBorder">
+        <Button onClick={onAddLink} variant="alternative" border="alternative">
```

Comment:
[NOTE] Looks like there is space for improvement in the future, depending on what other uses we will have for `<Button />`. For now it's good to be able to mix and match two props instead of one.